core-bundle/src/Resources/contao/models/PageModel.php

```diff
@@ -1015,6 +1016,8 @@ public function loadDetails()
 			$this->datimFormat = Config::get('datimFormat');
 		}
 
+		$this->isPublic = ($this->published && ($this->start == '' || $this->start <= $time) && ($this->stop == '' || $this->stop > ($time + 60)));
```

Comment:
I don't think this is a good idea. It's not a good idea to add such a prop in a patch version. Also `isPublic` could be *public cache*, it does not check user groups, or guest, so it could actually "not be public for the current visitor" etc.

Reply:
Well, we already have `$page->rootIsPublic`, therefore adding `isPublic` would follow the scheme of `alias` and `rootAlias`, `title` and `rootTitle` and so on.

Reply:
Valid point…